pkg/handler/scale_handler.go

```diff
@@ -392,6 +392,8 @@ func (h *ScaleHandler) getScaler(name, namespace, triggerType string, resolvedEn
 		return scalers.NewRedisStreamsScaler(resolvedEnv, triggerMetadata, authParams)
 	case "artemis-queue":
 		return scalers.NewArtemisQueueScaler(resolvedEnv, triggerMetadata, authParams)
+	case "http":
```

Comment:
I'd prefer not to call this `http` because people will think this allows them to scale HTTP workloads which is not the case.

What do you think @zroubalik @ahmelsayed? What about `external-metric-source`?

Reply:
Yeah, I wouldn't name it `http`. But I am not sure about the right name

Reply:
+1 for `external-metric-source`, I will add docs that desribe it nicely

Reply:
`external-metric-source` might give the impresion that this is type of  KEDA external scaler (which is not IMHO), but it might be just my perception 😄

Reply:
That's true, so how about `api-metric-source` to avoid any confusion with `external`?

Reply:
Sounds good to me! Or what about simply `metrics-api`?

Reply:
I am not good in naming, sorry guys 😄 `http-endpoint`, `api-endpoint` ... ?

Reply:
Let's use `metrics-api` 👍